### DISCUSSION TITLE
The start of wxmaxima Discussions on github

### DISCUSSION THREAD

**Discussion by gunterkoenigsmann** [wxMaxima-developers/wxmaxima#1446] [General]
Many users complained that there are too many wxMaxima sites in the net => I hope that adding a discussion forum here can get us rid of the last feature on the old sourceforge.net wxMaxima site.

**Comment by KubaO** [comment]
Good idea. Sourceforge is a disaster, whatever's left there should be "archived" so that the history persists but some general indication should be made that it's for historic interest only.

**Reply by gunterkoenigsmann** [reply]
If you have a sourceforge username I can make you an admin there: Until now I have been unable to find a good way to set the discussion to "read-only". But that might be caused by my amount of spare time being very limited, currently.

I'm still afraid that the owner of github one day tries to take all the magic they built for us away, but  Sourceforge's editor and the fact that the project is split into several pages aren't nice...

**Reply by zmth** [reply]
yea i see no reason maxima has to have anything to do with sourceforgenet. it just makes things much more confusing.

**Comment by KubaO** [comment]
Microsoft’s attitude towards open source has been radically changed on the recent years and they are open sourcing major feats of engineering, so I’d say that that’s unlikely. But you of course never know. I do have a SF user but have to figure out what it is and so on. Haven’t used it in years.

**Comment by gunterkoenigsmann** [comment]
Finally found a way to set the old forum to "read-only".

**Reply by Parlazo** [reply]
If the old forum is read only and the new discussion is here...
May the Forum-Link on the main-site should point to the discussion here and not to the old forum?
or at least a link to the new discussion here should get added to the main site there?
Just an idea
(https://wxmaxima-developers.github.io/wxmaxima/help.html)

**Reply by daute** [reply]
Added a link there as suggested.

**Comment by x-travis-x** [comment]
This might be a long shot, but I'm wondering if anyone knows of a way to disable/remove the Check for Updates function under the Help menu. I work in Higher Ed. and would like to remove the option. Thanks.

**Comment by KubaO** [comment]
There could be a configuration option for that, accessible from the configuration dialog. Would you consider that an acceptable solution? I could add that easily, and also a command line argument that would set this option without user interaction (ie. it could be given as a part of a launch shortcut).

**Reply by x-travis-x** [reply]
Oh sweet. I didn't think I'd get a response so quickly! I would accept either option as a solution. If it writes a regkey or a command line argument for all users I'm good with that. I am going to be exporting the regkey for the Tips on startup and applying it to the default user profile hive, so either way you feel is the best way to incorporate it. The launch shortcut could be an option, I'm build a Windows Virtual Desktop image (using RemoteApp) so multiple Faculty and Students will have access and I don't want anyone Checking for Updates. Thanks!

**Comment by gunterkoenigsmann** [comment]
I always wondered if we might want to remove that menu item altogether: even if one isn't behind a proxy and tries to click on this item and is informed that there is a new version: For MacOs one can only get new versions using the package manager, which is far behind. On Linux one normally gets new versions from the package manager, too. The official MS windows version lacks behind (am currently trying to create a github windows build for #1465 , but again fail to tell cmake where to find wxWidgets) so I always wondered who profits from the update check?

On the other hand if the update check is broken some users note this which might mean they actually intend to use this feature...

**Comment by KubaO** [comment]
I think I’ll do as follows:

1.  Add the configuration option.

2. Add the command line option that presets configuration option.

3. Add a cmake option to enable the update check.

4. Default the option to false.

For practical reasons, we’d really want this check to use a platform-specific repository, so whoever builds it should have the option of providing the URL, and perhaps a regex used to extract the version to compare, via cmake variables. This would allow distributions to choose an appropriate check, and if none is provided it would default to looking it up on GitHub releases.

So I’ll do 1-4, and see if doing the rest of it will be quick enough to warrant fixing it somewhat permanently.
